ha/heartbeat_test.go

```diff
@@ -35,11 +37,17 @@ func TestIcingaHeartbeatListener(t *testing.T) {
 	time.Sleep(time.Second * 2)
 
 	var uj interface{} = nil
-	if err := json.Unmarshal([]byte(icingastate), &uj); err != nil {
-		assert.Nil(t, err)
-	}
-
-	rdb.Rdb.Publish("icinga:stats", icingastate)
+	assert.Nil(t, json.Unmarshal([]byte(app), &uj))
+	assert.Nil(t, json.Unmarshal([]byte(dump), &uj))
+
+	rdb.Rdb.XAdd(&redis.XAddArgs{
```

Comment:
I think we should not use our own wrappers in this test. I know we did that before, but I don't think we should. Please remove our Publish and XAdd functions.